config/src/config/consensus_config.rs

```diff
@@ -15,6 +15,7 @@ pub struct ConsensusConfig {
     pub round_initial_timeout_ms: u64,
     pub proposer_type: ConsensusProposerType,
     pub safety_rules: SafetyRulesConfig,
+    pub stop_consensus: bool,
```

Comment:
I think we could probably have a detailed comment on when to use this option - given it is for extreme scenarios.  Essentially an operator wants to be available for syncing state but not commit any new transactions.  This is useful when validators coordinate on the latest version to apply a manual transaction or allow the network to synchronize the existing state without advancing it.

Also, stop_consensus is a bit ambiguous IMO.  Perhaps we could call it something like - sync_only or something?

Reply:
`sync_only` sounds a better name